viewer/view/IPCClient.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <QObject>
+#include <QUrl>
+
+namespace hdrv {
+  
+class IPCClient : public QObject
+{
+  Q_OBJECT
+
+public:
+  IPCClient() {}
```

Comment:
Probably, but then again, if you don't intend to use that constructor, why implement it?